spiceaidocs/content/en/reference/Spicepod/datasets.md

```diff
@@ -44,63 +44,55 @@ datasets:
       retention: 30m
 ```
 
-`spicepod.yaml`
-```yaml
-datasets:
-  - from: local/Users/phillip/data/test.parquet
-    name: test
-    acceleration:
-      enabled: true
-      mode: inmemory # / file
-      engine: arrow # / duckdb
-      refresh_interval: 1h
-      refresh_mode: full / append # update / incremental
-      retention: 30m
-```
-
 Relative path example:
 
 `spicepod.yaml`
 ```yaml
 datasets:
-  - from: datasets/uniswap_v2_eth_usdc
+  - from: datasets/eth_recent_transactions
 ```
 
-`datasets/uniswap_v2_eth_usdc/dataset.yaml`
+`datasets/eth_recent_transactions/dataset.yaml`
 ```yaml
-name: spiceai.uniswap_v2_eth_usdc
+from: spiceai:spice.ai/eth.recent_transactions
+name: eth_recent_transactions
 type: overwrite
-source: spice.ai
-auth: spice.ai
 acceleration:
   enabled: true
   refresh: 1h
 ```
 
-## `name`
+## `from`
 
-The name of the dataset. This is used to reference the dataset in the pod manifest, as well as in external data sources.
+The `from` field is a string that represents the Uniform Resource Identifier (URI) for the dataset. This URI is composed of two parts: a prefix indicating the source of the dataset, and the actual link to the dataset.
 
-## `type`
+The syntax for the `from` field is as follows:
 
-The type of dataset. The following types are supported:
+```yaml
+from: <source>:<link>
+```
 
-- `overwrite` - Overwrites the dataset with the contents of the dataset source.
-- `append` - Appends new data from dataset source to the dataset.
+Where:
+
+- `<source>`: The source of the dataset
 
-## `source`
+  Currently supported sources:
+  - `spiceai`
+  - `dremio`
+  - `databricks`
```

Comment:
s3 too.

Reply:
flightsql could be there in time, @Jeadie can update here

Reply:
Synced with available sources from https://github.com/spiceai/spiceai?tab=readme-ov-file#supported-data-connectors